app.py

```diff
@@ -35,7 +35,7 @@ def topics():
         response.headers.add('Access-Control-Allow-Origin', '*')
         return response
     except Exception as error:
-        return str(error)
+        return str(error), 500
```

Comment:
Great addition! Are we planning to add more errors (eg 404) in the future?

Reply:
we had 500 in as a catch all, but if you want to handle more error codes in the front we possibly could. 404s already handled if route is not found